README.md

```diff
@@ -12,11 +12,13 @@ This plugin is designed to be used on a per-field basis. To get started, add a s
 
 ### JSON Data Structure
 
-Each field requires a JSON configuration object. This object has 2 key fields — the first is `extends` and the second is `options`.
+Each field requires a JSON configuration object. This object has 2 key fields — the first is `extends` and the second is `preset`.
 
 To understand the behaviour of `extends`, see the [Global Presets](#global-presets) section below.
 
-`options` represents each visual option displayed to the editor, as well the underlying value returned by the API. There are 4 required fields on each option:
+`preset` Is an object with two keys - `options` and `presentation`
```

Comment:
My first fundamental question is why have you introduced a "preset" key to the JSON config? I think this new functionality would fit in simply via the "presentation" (optional) key present on the field config. That way, users who currently use the plugin won't require changes upon updating, as we can fall back to a default presentation mode (grid with six columns) if the "presentation" key<>object is missing. That would mean the final field JSON config would/can comprise three core keys, `presentation`, `extends`, and `options`.

Furthermore, the global config can and should remain unchanged as it's simply a source of reusable values that are presentation agnostic. For example, users could configure something like brand colours globally and on a per-field basis, decide to present it in the editor as a 6-column grid on one model but as a 3-column grid on another (not that I would personally do that, but it leaves the flexibility in the user's hands).

Reply:
Hey @tristan-go you're right, I introduced the key to be able to store presentations globally on a per-field basis, as doing that with an array of strings wouldn't be possible, so i made it into an object to have both the options and the presentation key inside the global field option, but as you said it is better to just delegate this to fields, adding a "presentation" key there, this way we can avoid much more complexity, and also completely skip the onBoot schema update. I did a new PR bellow with "Field presentations only" let me know what you think!

Reply:
Excellent, I'll look through all of it this morning and do a bit of testing.

@stefanoverna I agree that it's probably unlikely that the same global option set will be used on different models with different presentation settings. However, for now, locating the optional presentation settings on the field JSON config results in a much simpler and cleaner implementation, which should serve 95% of use cases. It's also backward compatible (when using a sensible presentation default) and requires no migration or mutation of existing JSON data structures — just to recap and confirm what's been discussed above :)